src/test/java/seedu/cakecollate/testutil/EditOrderDescriptorBuilder.java

```diff
@@ -80,9 +82,11 @@ public EditOrderDescriptorBuilder withAddress(String address) {
      * Sets the {@code OrderDescription} of the {@code EditOrderDescriptor} that we are building.
      */
     public EditOrderDescriptorBuilder withOrderDescriptions(String... orderDescriptions) {
-        Set<OrderDescription> orderDescriptionSet = Stream.of(orderDescriptions)
-                .map(OrderDescription::new).collect(Collectors.toSet());
-        descriptor.setOrderDescriptions(orderDescriptionSet);
+        Map<OrderDescription, Integer> orderDescriptionMap = new HashMap<>();
+        Stream.of(orderDescriptions)
+            .map(OrderDescription::new)
+            .forEach(o -> orderDescriptionMap.put(o, 1));
```

Comment:
Do you have to consider same order descriptions? like "for quantity times"

Reply:
Oh yeah, missed this out, thanks!